### ISSUE TITLE
How and where did you use it?

### ISSUE THREAD

**Issue by vkmavani** [jahongir7174/DeepLab-pt#1]
https://github.com/jahongir7174/DeepLab-pt/blob/b11ec2f34771f8cf7fd87f3debe3d6c38c401e54/nets/nn.py#L159

    def fuse(self):
        for n, m in self.named_modules():
            if type(m) is Conv and hasattr(m, 'norm'):
                if isinstance(m.relu, torch.nn.SiLU):
                    m.relu = SiLU(inplace=True)
                m.conv = fuse_conv(m.conv, m.norm)
                delattr(m, 'norm')
                m.forward = m.fuse_forward
            elif n == 'fc':
                m[1] = SiLU(inplace=True)
        return self

**Comment by vkmavani** [comment]
And also can you add a little explanation or full form of class names like "**ASPPModule**" and others. please...

**Comment by jahongir7174** [comment]
1. it is just for fusing batch norm into Conv layer, it is not used anywhere.
2. black rectangle is ASPPModule
![image](https://user-images.githubusercontent.com/71679734/115136713-99a38d00-a05c-11eb-85f5-7ca34decf812.png)

**Comment by vkmavani** [comment]
Thanks for quick reply.

**Comment by jahongir7174** [comment]
I am closing the issue, if you have any questions, you can open it again. Thank you
